reactioncommerce/catalyst
# feat: add confirm dialog

Resolves #17 
Impact: **major**  
Type: **feature**





## Component
- ConfirmDialog
- Add MUI theme

## Screenshots
--

## Breaking changes

none

## Testing
1. See the `ConfirmDialog` docs
2. Open the dialog to test
3. Click "cancel" once
4. Click "ok" once
